compiler/compile/OMRCompilation.hpp

```diff
@@ -1225,6 +1226,8 @@ class OMR_EXTENSIBLE Compilation
 
    BitVectorPool _bitVectorPool; //MUST be declared after _trMemory
 
+   bool _shouldCompile;
```

Comment:
Can we please Doxygen document what this field is used for? A naive reader such as myself may be wondering why someone may instantiate an `OMR::Compilation` object only to "not compile".